### PR TITLE
Removing Auth

### PR DESCRIPTION
**Pull request by vikasvmads**
Remove all code that requires auth to launch Core Editor.
Allow guest users to create files and download them.

### CODE REVIEW

File: src/CoreEditor/actions/publishActions.js [hunk 1]
```diff
@@ -136,16 +136,20 @@ export function publishDocumentSaveAsArticle() {
...
+      console.log(savePath);
       const html = exporter.articleDraftToHtml(
         editorState.getCurrentContent(),
         formatAuthor(wrioID),
         '',
         coverHtml ? coverSavePath : undefined,
         state.externalsEditor,
       );
-
-      const name = document && document.getProperty('name');
```
Comment (AlexeyAnshakov) [inline src/CoreEditor/actions/publishActions.js:147]: Why you deleted it?

File: src/CoreEditor/hello.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<html>
```
Comment (AlexeyAnshakov) [inline src/CoreEditor/hello.html]: We don't need this file

File: src/base/components/widgets/Login.js [hunk 3]
```diff
@@ -1,70 +1,70 @@
-import React from 'react';
-import Details from './Details.js';
-import { getServiceUrl, getDomain } from '../../servicelocator.js';
-import { Dropdown, MenuItem, Glyphicon } from 'react-bootstrap';
+// import React from 'react';
```
Comment (AlexeyAnshakov) [inline src/base/components/widgets/Login.js:1]: Why you deleted it?

File: src/base/components/widgets/Login.js [hunk 4]
```diff
@@ -1,70 +1,70 @@
...
+// import Details from './Details.js';
+// import { getServiceUrl, getDomain } from '../../servicelocator.js';
+// import { Dropdown, MenuItem, Glyphicon } from 'react-bootstrap';
```
Comment (AlexeyAnshakov) [inline src/base/components/widgets/Login.js:4]: Why you deleted it?

File: src/base/components/widgets/Login.js [hunk 5]
```diff
@@ -1,70 +1,70 @@
...
-    <i className="material-icons dp_big">account_circle</i>
-    Login
-  </a>
-);
+// const LoginButton = ({ onLogin }) => (
+//   <a href="#" className="btn btn-just-icon btn-simple btn-default btn-sm btn-flat pull-right">
+//     <i className="material-icons dp_big">account_circle</i>
+//     Login
+//   </a>
+// );
```
Comment (AlexeyAnshakov) [inline src/base/components/widgets/Login.js:13]: Let's left the login button. We will use it later.

File: src/base/components/widgets/Login.js [hunk 6]
```diff
@@ -1,70 +1,70 @@
...
-        !!readItLater.length && (readItLater.map((o, index) =>
-          <MenuItem href={o.url} key={index}>
-            <i className="material-icons dp_small with_text">bookmark</i>{o.name}
-          </MenuItem>
-        ))
-      }
-      {!!readItLater.length && (<MenuItem divider />)}
-      <MenuItem eventKey="1" href="https://core.wrioos.com/create">
-        <i className="material-icons dp_small with_text">create</i>Create new article
-      </MenuItem>
```
Comment (AlexeyAnshakov) [inline src/base/components/widgets/Login.js:47]: Let's it be. We still need to let guest user to create new articles

File: src/changes.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+Login.js  -
```
Comment (AlexeyAnshakov) [inline src/changes.js]: What does this file do?

File: src/devserver/hello.html [hunk 8]
```diff
@@ -0,0 +1,3 @@
+<html>
```
Comment (AlexeyAnshakov) [inline src/devserver/hello.html]: Delete

File: src/devserver/index.js [hunk 9]
```diff
@@ -36,6 +39,10 @@ coreService.get('/edit', (request, response) => {
   response.sendFile(`${__dirname}/core/core.html`);
 });
 
+app.get('/hello', (request, response) => {
+  response.sendFile(`${__dirname}/hello.html`);
+});
+
```
Comment (AlexeyAnshakov) [inline src/devserver/index.js]: Delete

File: src/iframes/Pinger/js/auth.js [hunk 10]
```diff
@@ -1,31 +1,31 @@
-/**
- * Created by michbil on 05.03.17.
- */
+// /**
+//  * Created by michbil on 05.03.17.
+//  */
```
Comment (AlexeyAnshakov) [inline src/iframes/Pinger/js/auth.js:3]: What does this change do?

File: src/CoreEditor/actions/publishActions.js (hunk 1 above)
Comment (vikasvmads) [reply src/CoreEditor/actions/publishActions.js:147]: Seems it's by mistake not forcefully.

File: src/CoreEditor/hello.html (hunk 2 above)
Comment (vikasvmads) [reply src/CoreEditor/hello.html]: It's a test file from my side. By mistake uploaded.

File: src/base/components/widgets/Login.js (hunk 5 above)
Comment (vikasvmads) [reply src/base/components/widgets/Login.js:13]: Ok sure.

I was thinking that it is not useful anymore.

File: src/changes.js (hunk 7 above)
Comment (vikasvmads) [reply src/changes.js]: It's a test file from my side. By mistake uploaded.

File: src/iframes/Pinger/js/auth.js (hunk 10 above)
Comment (vikasvmads) [reply src/iframes/Pinger/js/auth.js:3]: It was checking the auth. I assumed if we have guest users we don't need it.

File: src/devserver/index.js (hunk 9 above)
Comment (vikasvmads) [reply src/devserver/index.js]: Yes i will delete it.

File: src/base/components/widgets/Login.js (hunk 6 above)
Comment (vikasvmads) [reply src/base/components/widgets/Login.js:47]: Sure

File: src/CoreEditor/webrunesAPI.js [hunk 11]
```diff
@@ -35,8 +35,8 @@ export function getWidgetID(url: string): Promise {
     .withCredentials();
 }
 
-// export function getRegistredUser(): Promise {
-//   return request
-//     .get(`${protocol()}//login.${domain()}/api/get_profile`)
-//     .withCredentials();
-// }
+export function getRegistredUser(): Promise {
```
Comment (AlexeyAnshakov) [inline src/CoreEditor/webrunesAPI.js]: We have no working login.wrioos.com so it should be commented off. Just left the login button itself for later usage.

**Review by AlexeyAnshakov — CHANGES_REQUESTED** [review]

File: src/CoreEditor/webrunesAPI.js [hunk 12]
```diff
@@ -1,42 +0,0 @@
-/**
```
Comment (AlexeyAnshakov) [inline src/CoreEditor/webrunesAPI.js]: Why it's deleted? Maybe we don't need most of the functions now (btw, not sure about all) but we will need them later. Comment off unnecessary now.

File: src/base/components/FeedList.js [hunk 13]
```diff
@@ -122,7 +122,7 @@ class FeedListPage extends React.Component {
       console.log(error, "catch the hoop")
     })
   }
-  getSensorData() {
+  getSensorData = () => {
```
Comment (AlexeyAnshakov) [inline src/base/components/FeedList.js:125]: Why did you change this file? It's about another tasks

**Comment by vikasvmads** [comment]
@AlexeyAnshakov I have made the changes you suggested. Let me know if it's good.
